src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java

```diff
@@ -19,9 +19,8 @@
 import io.cdap.e2e.pages.locators.CdfBigQueryPropertiesLocators;
 import io.cdap.e2e.pages.locators.CdfStudioLocators;
 import io.cdap.e2e.utils.SeleniumHelper;
-
+import org.openqa.selenium.Keys;
```

Comment:
Is this required now? If not, please remove the unused import.

Reply:
done